packages/api-explorer/src/CodeSample.jsx

```diff
@@ -25,7 +25,10 @@ class CodeSample extends React.Component {
 
   getKey(example, index) {
     const key = `${example.language}-${index}`;
-    const selected = this.state.selectedExample === example;
+    let selected = this.state.selectedExample === example;
```

Comment:
Can you add some tests for this? I'm struggling to parse the logic behind this in my head, so having a test would be great!